### PR TITLE
Skeleton UI

### PR DESCRIPTION
**Pull request by oswynb**
List and delete functionality with paging for user-viewed tokens.
Placeholder CSS + 403/404 pages.

### CODE REVIEW

File: src/dummyui.hs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module Main where
+
+import Anchor.Tokens.Server.UI
+import System.Process
+import Control.Concurrent
+import System.IO
+
+main = do
+    writeFile "foo.html" dummy
+    system "chromium foo.html"
```
Comment (fractalcat) [inline src/dummyui.hs]: I'm not sure what the point of this bit is?

Comment (oswynb) [inline src/dummyui.hs]: something to just quickly view the dummy html

**Comment by thsutton** [comment]
Could maybe use https://hackage.haskell.org/package/file-embed to include CSS into the application? Given we aren't using templates and all.

**Comment by oswynb** [comment]
Is an idea, shall see what I can do

File: foo.ico [hunk 2]
```diff
@@ -0,0 +1 @@
...AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAATihYFkJCRgsOJgVYWWMGAAhdAWkaGC4uLiIiMTEBBgBaXVkBIiI0DF4mNC1kJxMAWkdZJhIYIiImYyYnM24hAAgsZUUJPDciD14MBjMzLwAVSVErXDg7DgsLDAwZOlAAJVUkKVdoagJPa14mPzE1AFQgJBFBQ1NdXk9MZ1MNWgBaMkpKUEgKK0BnAwYqKloAWjBHNmFsRj1SMS8TS2BaAFptRARNM1lTJy8QG2QIWgBaWmJhXRMjNCMtVhQ+WloAWlocEEoTOSoQbh1bW1RaAFpaWlofZDVUWloxXx5UWgBaWlpaZgdaWlpaWhYXWloAWlpaWmFaWlpaWlpaHFpaAAABAAAAAQAAAAEAAAABAAAAAQAAAAEAAAABAAAAAQAAAAEAAAABAAAAAQAAAAEAAAABAAAAAQAAAAEAAAABAAA=" rel="icon" type="image/x-icon" />
```
Comment (thsutton) [inline foo.ico]: Really? We're calling a fragment of HTML `.ico`?

File: lib/Anchor/Tokens/Server/API.hs [hunk 3]
```diff
@@ -29,9 +34,9 @@ import           Anchor.Tokens.Server.Types
 -- http://tools.ietf.org/html/rfc6749#section-3.1
 --
 -- @TODO(thsutton): This type should correctly describe the endpoint.
-type AuthorizeEndpoint
-    = "authorize"
-    :> Get '[JSON] [ByteString]
+--type AuthorizeEndpoint
```
Comment (thsutton) [inline lib/Anchor/Tokens/Server/API.hs]: Baleet plees

File: lib/Anchor/Tokens/Server/API.hs [hunk 4]
```diff
@@ -50,11 +61,25 @@ type VerifyEndpoint
...
+app :: Application
+app = serve api server
+  where
+    api :: Proxy AnchorOAuth2API
+    api = Proxy
 
 server :: Server AnchorOAuth2API
 server = error "Coming in Summer 2016"
+    :<|> error "Coming in Winter 2016"
+--    :<|> error "Coming in Spring 2016"
```
Comment (thsutton) [inline lib/Anchor/Tokens/Server/API.hs]: baleet plox

**Pull request merged**